index.html

```diff
@@ -8,7 +8,7 @@
       rel="stylesheet"
     />
     <link rel="stylesheet" href="styles.css" />
```

Comment:
- [ ] Hello 👋 please remember that according to the project requirements 👨‍🏫 you should check all of the items listed in the article [Easy Checks – A First Review of Web Accessibility](https://www.w3.org/WAI/test-evaluate/preliminary/) and make a report according to your findings. 

![image](https://user-images.githubusercontent.com/95243996/193809278-86919843-b124-4ddd-bc4e-14e7dbffc0c0.png)

Kyndly add your report to the description of your pull request this report can be something simple where you list all of the items you read on the article and describe your fingings.

Take this as an example.

![image](https://user-images.githubusercontent.com/95243996/193834665-3d0c54e1-ed09-441c-b7d6-26e4724c1a52.png)